Formula/jxrlib.rb

```diff
@@ -1,6 +1,6 @@
 class Jxrlib < Formula
   desc "Tools for JPEG-XR image encoding/decoding"
-  homepage "https://archive.codeplex.com/?p=jxrlib"
+  homepage "https://github.com/4creators/jxrlib"
```

Comment:
I think this is just a mirror. Since we're also using Debian for the source, let's also use Debian's tracker as the homepage.
```suggestion
  homepage "https://tracker.debian.org/pkg/jxrlib"
```

Reply:
👍